src/main/backend.ts

```diff
@@ -391,19 +379,16 @@ export class MainBackend implements CommonBackend {
       throw new SentryError('Invariant violation: Native crashes not enabled');
     }
 
-    const storedScope = Scope.clone(this.scope.get());
-    // tslint:disable:no-unsafe-any
-    const nextExtra = { ...storedScope.getExtra(), ...extra };
-    const event: SentryEvent = {
-      breadcrumbs: storedScope.getBreadcrumbs(),
-      extra: nextExtra,
-      tags: storedScope.getTags(),
-      user: storedScope.getUser(),
-    };
-    // tslint:enable:no-unsafe-any
+    const currentCloned = Scope.clone(getDefaultHub().getScope());
+    const fetchedScope = this.scopeStore.get();
+    (fetchedScope as any).eventProcessors = [];
```

Comment:
This looks a bit fishy. The stored scope should never have eventProcessors. It might be more defensive to remove the eventProcessors when setting into the store instead of when fetching it...